rcsb/app/file/mergeRequest.py

```diff
@@ -0,0 +1,74 @@
+import logging
+from pydantic import Field
+from pydantic import BaseModel  # pylint: disable=no-name-in-module
+from fastapi import Form
+from fastapi import Depends
+from fastapi import APIRouter
+from rcsb.utils.io.MarshalUtil import MarshalUtil
+from rcsb.utils.io.FileUtil import FileUtil
+from mmcif.io.PdbxWriter import PdbxWriter
+from rcsb.app.file.JWTAuthBearer import JWTAuthBearer
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+class MergeResult(BaseModel):
+    fileName: str = Field(None, title="Merged file name", description="Stored file name", example="1yy9_merged.cif")
+    success: bool = Field(None, title="Success status", description="Success status", example="True")
+    statusCode: int = Field(None, title="HTTP status code", description="HTTP status code", example="200")
+    statusMessage: str = Field(None, title="Status message", description="Status message", example="Success")
+
+
+@router.post("/merge", response_model=MergeResult, dependencies=[Depends(JWTAuthBearer())], tags=["merge"])
+async def merge(
+        siftsPath: str = Form(None),
+        pdbID: str = Form(None)
+):
+    ret = {}
+    try:
+        cachePath = "./rcsb/app/tests-file/test-data/mmcif/"
+        pdbIDHash = pdbID[1:3]
+
+        fU = FileUtil(workPath=cachePath)
+        if not fU.exists(cachePath):
+            fU.mkdir(cachePath)
+
+        mU = MarshalUtil(workDir=cachePath)
+
+        cifList = mU.doImport("https://ftp.wwpdb.org/pub/pdb/data/structures/divided/mmCIF/" + pdbIDHash + "/" + pdbID + ".cif.gz", fmt="mmcif")
```

Comment:
If the file is stored locally, (as defined in `cachePath` above, you shouldn't need to download it here.